Domain/Extensions/ExpressionExtension.cs

```diff
@@ -0,0 +1,18 @@
+using System.Linq.Expressions;
+
+namespace Domain.Extensions;
+
+public static class ExpressionExtension
```

Comment:
I don't agree, that this file should be on Domain side

Reply:
Application?

Reply:
for example yes